### PR TITLE
Move service performance dashboard into integrations

### PR DESCRIPTION
**Pull request by tijmenb**
## Context

This moves the dashboard to `/integrations/performance-dashboard`, to make it clearer it’s a special page that is open to the public. The service performance page now just links to it. This will make it easier to add non-public stuff to the support section in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1240, and prevents us making things public accidentally.

## Changes proposed in this pull request

See commit.

## Link to Trello card

https://trello.com/c/dKBwLMPv

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -324,6 +324,7 @@
   namespace :integrations, path: '/integrations' do
     post '/notify/callback' => 'notify#callback'
     get '/feature-flags' => 'feature_flags#index'
+    get '/performance-dashboard' => 'performance_dashboard#dashboard', as: :performance
```
Comment (benilovj) [inline config/routes.rb]: Do we want `dashboard` in the URL?

File: app/controllers/support_interface/performance_controller.rb [hunk 2]
```diff
@@ -1,9 +1,6 @@
 module SupportInterface
   class PerformanceController < SupportInterfaceController
-    skip_before_action :authenticate_support_user!, only: :index
-
     def index
```
Comment (stevehook) [inline app/controllers/support_interface/performance_controller.rb]: Would it make sense just to redirect to the new location?

File: config/routes.rb (hunk 1 above)
Comment (tijmenb) [reply config/routes.rb]: Yeah, I think that makes the purpose clearer.

File: app/controllers/support_interface/performance_controller.rb (hunk 2 above)
Comment (tijmenb) [reply app/controllers/support_interface/performance_controller.rb]: I'd like to add more stuff the the page in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1240, that's why we're keeping it and linking.

**Pull request merged**